new/LIT6948GenealogyMonks.xml

```diff
@@ -32,9 +32,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p>Genealogy of abbots of a monastery with the name <q xml:lang="gez">ደብረ፡ ሊባኖስ፡ </q> as it was noted in
-                    the incipit of <ref type="mss" corresp="FSUor40"/>. The name 'Dabra Libānos' can be identified with either 
-                    <placeName ref="INS0346DL"/> or <placeName ref="INS0341DL"/>.</p>
+                <p>Genealogy of the abbots of <placeName ref="INS0341DL"/> beginning with <persName ref="PRS9152TaklaHa"/>.</p>
```

Comment:
A genealogy is about what happened before, not after.
Genealogies usually go from St Anthony to Takla Haymanot, as in the passage you quoted in the initial issue.
Lists beginning with Takla Haymanot are not genealogies but lists of abbots. 
Is the manuscript you are describing with a genealogy or with a list of abbots?

Reply:
BLorient8814 has a genealogy from Anthony to Takla Haymanot (recorded as LIT3143Ledata now and pushed in a minute).
FSUor40 has a list of abbots to be recorded as LIT6948GenealogyMonks in future. I will change the title of the LIT record.

Reply:
No need to change the title of the LIT record, simply correct FSUor40 to refer to the existing CAe ID for the list of abbots

Reply:
> No need to change the title of the LIT record, simply correct FSUor40 to refer to the existing CAe ID for the list of abbots

Yes, I deleted LIT6948GenealogyMonks in a new commit.